data-prepper-plugins/key-value-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/keyvalue/KeyValueProcessor.java

```diff
@@ -94,6 +98,10 @@ public KeyValueProcessor(final PluginMetrics pluginMetrics, final KeyValueProces
         if(keyValueProcessorConfig.getIncludeKeys() != null) {
             includeKeysSet.addAll(keyValueProcessorConfig.getIncludeKeys());
         }
+
+        if(!validTransformOptionSet.contains(keyValueProcessorConfig.getTransformKey())) {
```

Comment:
nit: I believe convention is to have a space after `if` before parenthesis, to differentiate from function calls

i.e `if (!validTransform...) {`